lib/openid_connect/authorization/endpoint_start_point.rb

```diff
@@ -0,0 +1,28 @@
+module OpenidConnect
+  module Authorization
+    class EndpointStartPoint < Endpoint
+      def initialize(current_user)
+        super(current_user)
+      end
+      def handle_response_type(req, res)
```

Comment:
Unused method argument - `res`. If it's necessary, use `_` or `_res` as an argument name to indicate that it won't be used.<br>Use empty lines between defs.